src/porting.h

```diff
@@ -360,6 +360,9 @@ inline u32 getDeltaMs(u32 old_time_ms, u32 new_time_ms)
 	}
 #elif defined(_WIN32)
 	inline void setThreadName(const char* name) {}
+#elif defined(__GNU__)
+//#warning "GNU/Hurd platform, pthread_setname_np not yet supported"
+inline void setThreadName(const char* name) {}
```

Comment:
On Tue, 20. Jan 09:33 ShadowNinja <notifications@github.com> wrote:
>    In [1]src/porting.h:
>
>  > @@ -360,6 +360,9 @@ inline u32 getDeltaMs(u32 old_time_ms, u32 new_time_ms)
>  >    }
>  >  #elif defined(_WIN32)
>  >    inline void setThreadName(const char* name) {}
>  > +#elif defined(__GNU__)
>  > +//#warning "GNU/Hurd platform, pthread_setname_np not yet supported"
>  > +inline void setThreadName(const char* name) {}
>
>    This does effectively the same thing as the default. Also, either the
>    warning should be enabled or it should be a regular comment.

I suggest to keep the conditional because GNU/Hurd is now a distinct and
known platform and it might be possible that the behaviour will be
different from the default in the future. The comment can be omitted.